### PR TITLE
separated the PIM node into Azure AD and Azure Resources

### PR DESCRIPTION
**Pull request by CelesteDG**

### CODE REVIEW

File: api-reference/beta/toc.yml [hunk 1]
```diff
@@ -7865,9 +7865,10 @@
           href: api/policy-list-assigned.md
         - name: List applied to applications
           href: api/policy-list-appliesto.md
-      - name: Privileged Identity Management
-        href: resources/privilegedidentitymanagement-root.md
+      - name: Privileged Identity Management - Azure AD
```
Comment (dkershaw10) [inline api-reference/beta/toc.yml]: I looked at the review site, and this doesn't look so good.  The top level node look too long :(
Also the topics names underneath don't seem very consistent.

Comment (Lauragra) [reply api-reference/beta/toc.yml]: @dkershaw10 , what do you recommend? Possibly we can shorten to use PIM.

**Comment by Lauragra** [comment]
@CelesteDG , closing this one since we implemented the TOC update in a separate PR.

**Comment by CelesteDG** [comment]
Thank you, @Lauragra  :)

**Pull request closed without merging**